src/python/grpcio_tests/tests_aio/unit/channel_test.py

```diff
@@ -83,9 +91,10 @@ def test_unary_call_times_out(self):
                 self.assertIsNotNone(
                     exception_context.exception.trailing_metadata())
 
-        self.loop.run_until_complete(coro())
+        self._loop.run_until_complete(coro())
 
 
 if __name__ == '__main__':
+    aio.init_grpc_aio()
```

Comment:
this shouldn't be necessary since the test uses the `AioTestBase` which at each `setUp` tries to initialize the `Aio` package.

In any case having the feeling that `init_grpc_aio` is fundamentally broken since we are allowing to call it when there is no loop initialized, this might end up in a situation where the underlying gRPC library might make usage of a timer and not having any loop ready.

Having the feeling that we would need to change the `init_grpc_aio` function, by either starting a loop or asking for a loop as a parameter and checking that the loop is running.

Reply:
Removed for all test files.

Yes. That is indeed an issue, I created https://github.com/grpc/grpc/issues/20806 to track it, so we won't forget to change it.